### PR TITLE
blackmagic decklink sink

### PR DESCRIPTION
**Pull request by funman**

### CODE REVIEW

File: lib/upipe-blackmagic/sdi.c [hunk 1]
```diff
@@ -0,0 +1,263 @@
...
+
+    dst[ANC_START_LEN + 9 + src_size + 3] = checksum ? 256 - checksum : 0;
+
+    dst[DC_POS] = cnt; // DC
+}
+
+static inline uint32_t to_le32(uint32_t a)
+{
+#ifdef UPIPE_WORDS_BIGENDIAN
+    return __builtin_bswap32(a);
```
Comment (cmassiot) [inline lib/upipe-blackmagic/sdi.c]: I guess you will need a non-gcc implementation again.

File: lib/upipe-blackmagic/sdi.c [hunk 2]
```diff
@@ -0,0 +1,263 @@
+#include <stdbool.h>
```
Comment (cmassiot) [inline lib/upipe-blackmagic/sdi.c]: IIUC, there is nothing blackmagic-specific in this module, perhaps it should be moved to lib/upipe or upipe-modules ?

Comment (funman) [reply lib/upipe-blackmagic/sdi.c]: True, although not sure what else could use it, maybe vanc filter?

Comment (cmassiot) [reply lib/upipe-blackmagic/sdi.c]: Another SDI output module (dektec or deltacast for instance).

**Comment by cmassiot** [comment]
Do you want to check it in as is ?

**Comment by funman** [comment]
Why not, it works fine as-is.

We can always work on it later when someone add output for another card

**Comment by cmassiot** [comment]
So can you squash it ?

**Pull request merged**